testsuite/cluster-test/src/experiments/twin_validator.rs

```diff
@@ -44,7 +44,7 @@ impl ExperimentParam for TwinValidatorsParams {
         let mut twin_validators = vec![];
         let mut rnd = rand::thread_rng();
         for _i in 0..self.pair {
-            twin_validators.push(instances.remove(rnd.gen_range(0, instances.len())));
+            twin_validators.push(instances.remove(rnd.gen_range(1, instances.len())));
```

Comment:
Need to update `self.pair > 3` - it is a bit strange now.
Need to check that number of pairs is less then instances.len-1